pkg/controllers/repositoryserver/repository.go

```diff
@@ -52,3 +57,20 @@ func (h *RepoServerHandler) connectToKopiaRepository() error {
 		args,
 	)
 }
+
+func (h *RepoServerHandler) getRepositoryCacheSettings() (contentCacheMB, metadataCacheMB int, err error) {
+	contentCacheMB, metadataCacheMB = command.GetGeneralCacheSizeSettings()
+	if h.RepositoryServer.Spec.Repository.CacheSizeSettings.Content != "" {
```

Comment:
What happens here when the CR does not have `CacheSizeSettings`? Would it resolve to `{Content: "", Metadata: ""}`?

Reply:
yes @pavannd1